crypto_test.go

```diff
@@ -13,6 +13,7 @@ import (
 )
 
 // These values come from tenant-security-client-php for easy cross-SDK testing.
+// nolint:lll
```

Comment:
```suggestion
//nolint:lll
```

Getting rid of the whitespace prevents the `nolint` comment from showing up in Go Docs.